community/examples/slurm-gcp-v5-high-io.yaml

```diff
@@ -0,0 +1,135 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: hpc-cluster-high-io-v5
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: high-io-slurm-gcp-v5
+  region: us-central1
+  zone: us-central1-c
+  enable_reconfigure: true
+
+# Documentation for each of the modules used below can be found at
+# https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/modules/README.md
+
+deployment_groups:
+- group: primary
+  modules:
+  # Source is an embedded module, denoted by "modules/*" without ./, ../, /
+  # as a prefix. To refer to a local or community module, prefix with ./, ../ or /
+  # Example - ./modules/network/pre-existing-vpc
+  - id: network1
+    source: modules/network/vpc
+
+  - id: homefs
+    source: modules/file-system/filestore
+    use: [network1]
+    settings:
+      local_mount: /home
+
+  - id: projectsfs
+    source: modules/file-system/filestore
+    use: [network1]
+    settings:
+      filestore_tier: HIGH_SCALE_SSD
+      size_gb: 10240
```

Comment:
While testing this I discovered that the Terraform provider does not fail when Cloud Logging is littered with failures such as:

```
Quota limit 'HighScaleSSDStorageGibPerRegion' has been exceeded. Limit: 0 in region us-central1.
```

It kept going for 37 minutes. I ^C'd out once... waited anothe 90 seconds and ^C'd again.

At that point i could no longer run `terraform destroy` due to "invalid index" errors:

```
 Error: Invalid index
│
│   on modules/schedmd-slurm-gcp-v5-controller/outputs.tf line 19, in output "controller_instance_id":
│   19:   value       = module.slurm_controller_instance.slurm_controller_instance.instances_details[0].id
│     ├────────────────
│     │ module.slurm_controller_instance.slurm_controller_instance.instances_details is empty tuple
│
│ The given key does not identify an element in this collection value: the collection has no elements.
╵
╷
│ Error: Invalid index
│
│   on .terraform/modules/slurm_controller.slurm_controller_instance/terraform/slurm_cluster/modules/slurm_controller_instance/main.tf line 351, in module "slurm_pubsub":
│  351:       name                    = module.slurm_controller_instance.instances_details[0].name
│     ├────────────────
│     │ module.slurm_controller_instance.instances_details is empty tuple
│
│ The given key does not identify an element in this collection value: the collection has no elements.
╵
╷
│ Error: Invalid index
│
│   on .terraform/modules/slurm_controller.slurm_controller_instance/terraform/slurm_cluster/modules/slurm_controller_instance/main.tf line 510, in module "reconfigure_critical":
│  510:       controller_id = module.slurm_controller_instance.instances_details[0].instance_id
│     ├────────────────
│     │ module.slurm_controller_instance.instances_details is empty tuple
│
│ The given key does not identify an element in this collection value: the collection has no elements.
```

This most likely reflects from fragility in the upstream module, but I think we might consider removing the HIGH_SCALE_SSD from the example until we resolve the bug.

I realize you're mimicking the existing example for the v4 modules, but this is not a good introduction to Filestore High Scale.

Reply:
The failure to destroy should be fixed by #739 
The issue with quota and timing out only after an hour is not specific to this example, so therefore I believe this PR should still go through after #739 is merged.

Reply:
I agree, we should still keep it.

Reply:
#739  has been merged, the destroy failure should be fixed.